make/Makefile.dev

```diff
@@ -0,0 +1,60 @@
+DOCKER_REPO?=quay.io/openshiftio
+IMAGE_NAME?=git-service
+SHORT_COMMIT=$(shell git rev-parse --short HEAD)
+ifneq ($(GITUNTRACKEDCHANGES),)
+SHORT_COMMIT := $(SHORT_COMMIT)-dirty
+endif
+
+TIMESTAMP:=$(shell date +%s)
+TAG?=$(SHORT_COMMIT)-$(TIMESTAMP)
+
+DEPLOY_DIR:=deploy
+
+.PHONY: create-resources
+create-resources:
+	@echo "Logging using system:admin..."
+	@oc login -u system:admin
+	@echo "Creating sub resources..."
+	@echo "Creating CRDs..."
+	@oc create -f https://raw.githubusercontent.com/redhat-developer/devopsconsole-operator/master/deploy/crds/devopsconsole_v1alpha1_gitsource_crd.yaml
```

Comment:
or we could include the `redhat-developer/devopsconsole-operator/deploy/crds` package as a required dependency (using `dep`) and create the CRD from the file in the `vendor` directory. It could bring more stability compared to always applying the latest version from `master` branch on GitHub.

Reply:
I like the idea. But it looks like we can't require a dependency with no go code.
See https://github.com/golang/dep/issues/1306

Reply:
ah right. As a workaround, we could have a `doc.go` file in that package, maybe?